server/api/images.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import Router from 'koa-router';
+import auth from '../middlewares/authentication';
+import { getSignedImageUrl } from '../utils/s3';
+
+const router = new Router();
+
+router.post('images.info', auth(), async ctx => {
```

Comment:
```suggestion
router.get('images.info', auth(), async ctx => {
```

This is one route where only `get` makes sense, I think.

Reply:
I had it as a `get` initially, but it wasn't working. After digging, think it could be because of the `methodOverride` middleware, so I just switched it to `post`. After the initial mystery, I quite liked the uniformity of having everything transformed to a `post`! Do you know a way to override the `methodOverride` here, or is it a case of a whole new `Koa()` instance?

Reply:
Got it, we can leave then 👍